src/lib/queryClient.ts

```diff
@@ -4,6 +4,7 @@ const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
       suspense: true,
+      useErrorBoundary: true,
```

Comment:
suspense 속성을 true로 한다면 useErrorBoundary는 기본적으로 true로 설정되서 생략해도 상관없을 것 같습니다!

>By default, instead of supplying the error variable when a mutation fails, it will be thrown during the next render of the component it's used in and propagate to the nearest error boundary, similar to query errors.
https://react-query-v3.tanstack.com/guides/suspense#_top

Reply:
몰랐던부분인데 감사합니다👍 역시 공식문서를 항상 잘봐야하네요😂